examples/resources/cockroach_api_key/import.sh

```diff
@@ -0,0 +1,4 @@
+# Since the secret, is not retreivable after creation, it must be provided
+# during import.  The API key ID can be derived from the secret.
```

Comment:
Why/how is the API key ID derivable from the secret? @fantapop

Reply:
The secret contains the api key id as a prefix.  I don't know the reasons this was done but it makes it easy to determine if your secret matches a specific key ID which is handy.